CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
+- Fixed alignment issue with sidebar and table of contents alignment. [#556](https://github.com/blackbaud/stache2/pull/556)
+- Updated version of `skyux-builder`.
+- Updated Travis build options to enhance test utility.
+- Updated skyux dependencies to depend on components directly. [#553](https://github.com/blackbaud/stache2/pull/553)
```

Comment:
"skyux" should be "SKY UX"

FWIW, "dependencies to depend on" seems awkward. And I'm not sure what we mean by it.